test/nodejs/test_file.rb

```diff
@@ -54,6 +54,26 @@ def test_read_each_line
       lines << line
     end
     assert_equal(lines.length, 2)
+    assert_equal(lines[0], 'one')
+    assert_equal(lines[1], 'two')
+  end
+
+  def test_readlines
+    File.write('tmp/quz', "one\ntwo")
+    file = File.new('tmp/quz', 'r')
+    lines = file.readlines do |line|
```

Comment:
There's a `SyntaxError`, this block doesn't have `end`

Reply:
fixed